### DISCUSSION TITLE
Future of case studies section

### DISCUSSION THREAD

**Discussion by sftim** [kubernetes/website#48432] [Ideas]
When Kubernetes was relatively new, we (and the nascent CNCF) promoted it via [case studies](https://k8s.io/case-studies/).

These case studies are (as of 2024-10-19) relatively old and there are no current plans to add new ones.

Should we keep this section? If we do, how will we keep it current?

---

See PR https://github.com/kubernetes/website/pull/57043

**Comment by shurup** [comment]
Earlier in Slack, I expressed that we no longer need case studies today. However, my recent observations led me to an opposite conclusion. Many of us live in a bubble where "Kubernetes is an absolute [and the only] winner," yet even today in the bigger IT community, you still find many concerns about whether we should use it for various cases. That happened to me due to some migration-from-Kubernetes articles popping out and comparing the reaction to pro-Kubernetes posts in different tech communities (many IT folks are still not into Kubernetes).

As I just discovered, the CNCF website has a [case studies section](https://www.cncf.io/case-studies) that is continuously updated with new cases and has a technology filter that allows you to show [Kubernetes-related cases](https://www.cncf.io/case-studies?_sft_lf-project=kubernetes). Currently, it has cases featuring Kubernetes dated January 2025, December 2024, etc.

Based on that*, I think we can keep this section on the Kubernetes website and (from the easiest to the most advanced solution):

1. Add a noticeable link to this page stating that here's where you can find newer cases.
2. Periodically and manually select a few cases (e.g., those that significantly rely on Kubernetes) and add their brief descriptions to the website with links to the CNCF website for more details.
3. Implement automation to bring new cases to the Kubernetes website from there.

**Comment by sftim** [comment]
- We could change the top nav to link directly to https://www.cncf.io/case-studies?_sft_lf-project=kubernetes (maybe marking it somehow as an external link).
- We could automate copying selected case studies to the foot of https://k8s.io/

**Comment by SayakMukhopadhyay** [comment]
> We could change the top nav to link directly to

Does the CNCF case study page have all the case studies on the k8s website?

**Reply by lmktfy** [reply]
Even if it doesn't, I personally recommend that we switch over to having only CNCF-hosted case studies.

**Comment by lmktfy** [comment]
Case studies are now not linked from the primary nav (but they are linked from the landing page and from the community page).

**Comment by lmktfy** [comment]
We are proposing to remove case studies - see https://github.com/kubernetes/website/pull/57043
